### PR TITLE
[GLOSSARY] blockchain

### PR DESCRIPTION
**Pull request by MaxHillebrand**
adds timechain to glossary

### CODE REVIEW

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -118,6 +118,14 @@ The method is used by some lightweight Bitcoin clients.
 Read more: [Wasabi Wallet under the hood](/building-wasabi/TechnicalOverview.md#wasabi-wallet-under-the-hood)
 :::
 
+:::details
+### Timechain
+
+The timechain is the one chain of valid Bitcoin blocks with the most accumulated proof of work.
+It is an append only data structure with new blocks being added on average every 10 minutes.
```
Comment (SimoneBovi) [inline docs/glossary/Glossary.md]: It is an append-only data structure with new blocks being added on average every 10 minutes.

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
It is an append-only data structure with new blocks being added on average every 10 minutes.
```

**Comment by RiccardoMasutti** [comment]
NACK.
I think Bitcoin basics should not be in Wasabi glossary.

**Comment by yahiheb** [comment]
Shouldn't we say it is synonym to `Blockchain`?

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -118,6 +118,14 @@ The method is used by some lightweight Bitcoin clients.
 Read more: [Wasabi Wallet under the hood](/building-wasabi/TechnicalOverview.md#wasabi-wallet-under-the-hood)
 :::
 
+:::details
+### Timechain
+
+The timechain is the one chain of valid Bitcoin blocks with the most accumulated proof of work.
+It is an append-only data structure with new blocks being added on average every 10 minutes.
```
Comment (MaxHillebrand) [inline docs/glossary/Glossary.md]:
```suggestion
It is an append-only data structure with new blocks being added on average every 10 minutes, thus some call it blockchain.
```

Comment (yahiheb) [reply docs/glossary/Glossary.md]: ACK. I think most call it blockchain.

Comment (SimoneBovi) [reply docs/glossary/Glossary.md]: I agree with @yahiheb. Related to that, I think it could be confusing to use the term timechain on the Doc homepage.

**Comment by yahiheb** [comment]
Maybe we should replace `Timechain` by `Blockchain` and mention that some people call it `Timechain`.

**Comment by MaxHillebrand** [comment]
`blockchain` is mentioned 15 + times in the docs, thus it should be included in the glossary.

**Pull request merged**